plugin/restore/mysql/mysql.go

```diff
@@ -4,11 +4,20 @@ import (
 	"bufio"
 	"context"
 	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
 	"strings"
+	"time"
 
-	"github.com/bytebase/bytebase/plugin/db/mysql"
+	"github.com/bytebase/bytebase/api"
+	mysql "github.com/bytebase/bytebase/plugin/db/mysql"
 )
 
+// TODO(zp): refactor this when sure the mysqlbinlog path
+var mysqlbinlogPath = "UNKNOWN"
```

Comment:
It's the path of mysqlbinlog binary instead of binlog path, binlog path is an argument of `SyncBinlog()`.
I will set it in another pr. I wanna extract it in booting bytebase logic like Postgres.

Reply:
Sorry for the mistake, I was talking about the binlog file saving dir.
But this is still misleading, maybe change to `mysqlbinlogBinPath`?

Reply:
You once corrected `mysqlbinlogbinpath` to `mysqlbinlogpath`. But that's ok.